webpack.common.js

```diff
@@ -0,0 +1,43 @@
+const path = require("path");
```

Comment:
The new CI setup using webpack and babel. This is being used in `package.json`'s npm scripts. An update to our current `README` is needed.